packages/gatsby-plugin-sharp/src/image-data.ts

```diff
@@ -256,6 +256,7 @@ export async function generateImageData({
         height: Math.round(width / imageSizes.aspectRatio),
         toFormat: `avif`,
       })
+      if (pathPrefix) transform.pathPrefix = pathPrefix
```

Comment:
nit: I'm a big fan of adding braces, even when they're not needed. Makes it less easy to break, and easier to read

Reply:
Fair enough :)